portfolio/src/main/webapp/index.html

```diff
@@ -53,14 +53,20 @@ <h3 style="background-color:WhiteSmoke; color:DimGrey">Comment Box</h3>
         <textarea name="comments-input" id="textbox"></textarea><br/>
         <p>Please insert the maximum number of comments that you want to see:</p>
         <input type="number" name="max-comments-num" min="1" value="3"><br/><br/>
-        <input type="submit" value="Submit" />
+        <input type="submit" value="Submit">
       </form>
-      <h3 style="background-color:WhiteSmoke; color:DimGrey">Comments Collection</h3>
+      <h3 style="background-color:WhiteSmoke; color:DimGrey">Image Upload</h3>
+      <form id="my-form" class="hidden" method="POST" enctype="multipart/form-data">
```

Comment:
Nit: this form id is quite generic - better to name it something related to its use. Perhaps `image-upload-form`?

Reply:
Yep. I have fixed that! Thank you for the advice.